chatapi_ros/launch/chatapi.launch

```diff
@@ -0,0 +1,58 @@
+<launch>
+
+  <arg name="use_chaplus" default="true" />
+  <arg name="use_a3rt" default="true" />
+
+  <arg name="launch_sound_play" default="false" />
```

Comment:
if this is copy of https://github.com/jsk-ros-pkg/jsk_3rdparty/blob/master/ros_speech_recognition/launch/speech_recognition.launch, please use `<include`. If there is some updates, please send PR.